content/docs/dvclive/live/log_sklearn_plot.md

```diff
@@ -192,18 +192,23 @@ $ dvc plots show 'dvclive/plots/sklearn/roc.json' \
 
 ## Parameters
 
-- `kind` - a [supported plot type](#supported-plots)
+- `kind` - a [supported plot type](#supported-plots).
 
-- `labels` - array of ground truth labels
+- `labels` - array of ground truth labels.
 
 - `predictions` - array of predicted labels (for `confusion_matrix`) or
-  predicted probabilities (for other plots)
+  predicted probabilities (for other plots).
 
-- `name` - Optional name of the output file. If not provided, `kind` will be
+- `name` - optional name of the output file. If not provided, `kind` will be
   used as name.
 
-- `**kwargs` - additional arguments to be passed to the internal scikit-learn
-  function being called
+- `**kwargs` - additional arguments to tune the result. Arguments are passed to
+  the scikit-learn function (e.g. `drop_intermediate=True` for the
+  [`roc`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html)
+  type). Plus extra arguments supported by the type of a plot are:
+
+  - `normalized` - _default_: `False`. `confusion_matrix` with values normalized
+    to `<0, 1>` range.
```

Comment:
Didn't want to block the code P.R. as it is not critical but this highlights what I consider a problem which is that `confusion_matrix` should not really be part of `log_sklearn_plot`, as it doesn't really call `sklearn` under the hood.

Reply:
Should we move confusion matrix to https://github.com/iterative/dvclive/issues/271 once we implement that? We could make a new issue or note it there.

Reply:
> Should we move confusion matrix to https://github.com/iterative/dvclive/issues/271 once we implement that? We could make a new issue or note it there.

Yep. Would probably need to keep it hidden but working until the next major release.

I have just started working on #271 , it looks simple enough but useful to showcase now that we are pushing for DVCLive more